config/settings/base.py

```diff
@@ -137,3 +137,6 @@
 STATIC_ROOT = join(ROOT_DIR, "staticfiles")
 STATIC_URL = "/static/"
 COMPRESS_ENABLED = True
+STATICFILES_STORAGE = "django.contrib.staticfiles.storage.ManifestStaticFilesStorage"
+
+CACHES = {"default": env.cache("CACHE_URL", default="locmemcache://")}
```

Comment:
I'm assuming we'll use redis in Prod?

Reply:
Yes, this allows us to. Not sure if it's necessary, but we'll be able to.